### ISSUE TITLE
"Add work email address" button doesn't show up and text copy is not the correct one 

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#6439]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Create new gmail account on New Expensify App (Build 1.1.16-1)
2. Add an "OPEN" Bank Account
3. Verify the bank account is added with a button to add work email address

## Expected Result:
Bank Account added and a button to add work email address. Also the copy of the text reads: 
![image](https://user-images.githubusercontent.com/44479856/143090564-30d9fa3c-b2c5-4623-bdfd-81e99fa6eb33.png)

## Actual Result:
The Bank Account is added but the button to add work email address doesn't show up. A button with "Chat with Concierge" appears instead. Also copy of the text is not the correct one, it reads: 
![image](https://user-images.githubusercontent.com/44479856/143090652-74d562a2-3d7f-4e3a-95f1-f241e8b4f25f.png)

## Workaround:
No workaround to finish the PR steps https://github.com/Expensify/App/pull/6319 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.16-2

**Reproducible in staging?:** Yes 
**Reproducible in production?:** N/A (New Feature) 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![image](https://user-images.githubusercontent.com/44479856/143090413-f00b1955-4067-4e2f-b84a-88e92a6b0153.png)

**Expensify/Expensify Issue URL:**

**Issue reported by:**
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @iwiznia (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by isagoico** [comment]
Issue is failing PR https://github.com/Expensify/App/pull/6319

**Comment by luacmartins** [comment]
Discussing it here https://expensify.slack.com/archives/C03TQ48KC/p1637699458093500

**Comment by luacmartins** [comment]
Even though the PR was deployed to staging, looking at the `staging` branch, the code doesn't seem to be there. We are CP'ing the changes again and we can try QA'ing it once more.

**Comment by luacmartins** [comment]
Code is on staging, @isagoico would you mind retesting https://github.com/Expensify/App/pull/6319 please? If all looks good we can check it off the list.

**Comment by luacmartins** [comment]
Actually, I just tested it on staging and all looks good! Closing this issue and checking it off the list. Thanks!

<img width="1018" alt="Screen Shot 2021-11-23 at 4 03 01 PM" src="https://user-images.githubusercontent.com/22219519/143142802-c95011d2-c987-442f-8366-c970f442fe8f.png">

<img width="974" alt="Screen Shot 2021-11-23 at 4 06 40 PM" src="https://user-images.githubusercontent.com/22219519/143142880-e80fd9b9-ba0b-4590-81be-376e1596016a.png">
